source/opt/def_use_manager.cpp

```diff
@@ -38,10 +38,14 @@ namespace analysis {
 
 void DefUseManager::AnalyzeInstDefUse(ir::Instruction* inst) {
   const uint32_t def_id = inst->result_id();
-  // Clear the records of def_id first if it has been recorded before.
-  ClearDef(def_id);
+  if (def_id != 0) {
+    ClearDef(def_id);
```

Comment:
It seems this is the only use of `ClearDef`? Actually we can use `ClearInst(inst)` here directly so that we can remove `ClearDef`?